usaspending_api/common/api_request_utils.py

```diff
@@ -309,6 +311,79 @@ def validate(body):
                 raise Exception("Invalid mode, autocomplete modes are 'contains', 'startswith', but got " + body["mode"])
 
 
+class GeoCompleteHandler:
+    """ Handles geographical hierarchy searches """
+
+    def __init__(self, request_body):
+        self.request_body = request_body
+        self.search_fields = {
+          "location_country_code__country_name": {
+            "type": "COUNTRY",
+            "parent": "location_country_code"
+          },
+          "location_state_code": {
+            "type": "STATE",
+            "parent": "location_country_code__country_name"
+          },
+          "location_state_name": {
+            "type": "STATE",
+            "parent": "location_country_code__country_name"
+          },
+          "location_city_name": {
+            "type": "CITY",
+            "parent": "location_county_name"
```

Comment:
It doesn't inform how it currently works, no. 
I don't think multi-county cities will impact anything bizarrely, it will just function off of whatever county is specified by the Broker, or is found in RefCityCountyCodes